kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/orders/data/User.java

```diff
@@ -0,0 +1,97 @@
+package com.kodilla.good.patterns.orders.data;
+
+public class User {
+    private final int id;
+    private final String name;
```

Comment:
why do you have firstName, lastName and name fields?

Reply:
I'll exchange `name` for `nickname`. It'll be better.